roles/dhcp-server/templates/local.conf.j2

```diff
@@ -8,3 +8,6 @@ dhcp-boot=pxelinux.0
 dhcp-authoritative
 enable-tftp
 tftp-root=/srv/tftp
+no-hosts
```

Comment:
If we're using our static hosts stuff, then we'll probably want to serve those over DNS too. wouldn't we?